content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,305 @@
+---
+{
+    title: "Introduction to Web Accessibility (A11Y)",
+    description: "",
+    published: '2021-05-30T22:12:03.284Z',
+    authors: ['crutchcorn'],
+    tags: ['a11y', 'javascript', 'web'],
```

Comment:
include accessibility, 508 tags
```suggestion
    tags: ['a11y', 'javascript', 'web', accessibility, 508],
```

Reply:
Wait, what's 508?

Reply:
https://www.section508.gov/manage/laws-and-policies
the US laws regulating a large portion of  a11y